### DISCUSSION TITLE
RC versioning

### DISCUSSION THREAD

**Discussion by pellared** [open-telemetry/opentelemetry-dotnet#3039] [General]
Today, I noticed that for both NuGet and dependabot `1.0.0-rc9 > 1.0.0-rc10`. See:
- https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/410
- https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AspNet/ (see versions tab)

Versioning `1.0.0-rc.9` and `1.0.0-rc.10` would work better

It works for NuGet: https://www.nuget.org/packages/Astralis.Domain/

Someone could test if it works for dependabot as well. However, even if it won't, then I do not think we can do anything better.

I think this pattern could be used in the future.

**Comment by cijothomas** [comment]
https://github.com/open-telemetry/opentelemetry-dotnet/issues/2991
